WebApi/WebApiTest/ControllerTest/CommentControllerTest.cs

```diff
@@ -0,0 +1,238 @@
+﻿using Microsoft.Extensions.Logging;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WebApi.Controllers;
+using WebApi.Models.DTO;
+using WebApi.Services.Services_Interfaces;
+using Xunit;
+
+namespace WebApiTest.ControllerTest
+{
+    public class CommentControllerTest
+    {
+        const int UserId = 1;
+        [Theory]
+        [InlineData(1, 1, 1, "test")]
+        [InlineData(2, 2, 2, "test2")]
+        [InlineData(3, 3, 3, "test3")]
+
+        public void GetById_Test(int c_id, int u_id, int p_id, string content)
+        {
+            DateTime date = new DateTime(2008, 3, 1, 7, 0, 0);
```

Comment:
Domyślam się że jest problem żeby tą datę dodać do inline data bo sam nie mogłem znaleźć jak to zrobić, ale myślę że taki test wystarczy